src/index.jsx

```diff
@@ -1,13 +1,18 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
-import './i18n/setup';
+import { I18nextProvider } from 'react-i18next';
+import i18n from './i18n';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Suspense fallback="loading">
```

Comment:
Is it possible to have no fallback or the empty string? I just tried simulating a slow 3G connection and it took like 20 seconds before this showed up and then another 5 seconds for the rest of the home page to load.

Reply:
without the `Suspense` I get 

"Error: Header suspended while rendering, but no fallback UI was specified.
Add a <Suspense fallback=...> component higher in the tree to provide a loading indicator or placeholder to display."

Reply:
I changed it to an empty string instead